src/components/EditBorderDialog.vue

```diff
@@ -1,24 +1,25 @@
 <template>
```

Comment:
Why is edit border dialog in here ? I thought we removed these changes on this branch